packages/nc-gui/components/smartsheet/column/LinkedToAnotherRecordOptions.vue

```diff
@@ -74,16 +74,18 @@ const isLinks = computed(() => vModel.value.uidt === UITypes.Links)
           v-model:value="vModel.childId"
           show-search
           :filter-option="filterOption"
-          dropdown-class-name="nc-dropdown-ltar-child-table"
+          dropdown-class-name="nc-dropdown-ltar-child-table "
```

Comment:
Extra space here. Intentional?

Reply:
no i thinks its prettier

Reply:
**fixed in below Commit**

[fix : removed unwanted ! - important](https://github.com/nocodb/nocodb/pull/7114/commits/b862466785ff4be99629740e90baa00bb3b14731)
[b862466](https://github.com/nocodb/nocodb/pull/7114/commits/b862466785ff4be99629740e90baa00bb3b14731)